src/lib/Liblog/pbs_log.c

```diff
@@ -137,6 +138,8 @@ static unsigned int syslogfac = 0;
 static unsigned int syslogsvr = 3;
 static unsigned int pbs_log_highres_timestamp = 0;
 
+extern char *pbs_asprintf_format(int len, const char *fmt, va_list args);
```

Comment:
I suggest putting this in libutil.h and remove from here as well as log_event.c (and include libutil.h if necessary)